packages/nc-gui/components/ProjectTreeView.vue

```diff
@@ -593,6 +593,17 @@
             </settings-modal>
           </div>
         </template>
+
+        <div
+          v-t="['api-docs']"
+          class="caption pointer nc-team-settings pb-3 pl-5 pr-3 pt-2 d-flex align-center"
+          @click="openLink('https://apis.nocodb.com')"
```

Comment:
Should we show the all-apis one? Some users may be interested in APIs like how to create a table.

Reply:
Lets point it to apis.nocodb for now.

All APIs will be overwhelming for user who doesnt need the meta apis